lib/matplotlib/animation.py

```diff
@@ -463,6 +483,12 @@ def cleanup(self):
 # Base class of ffmpeg information. Has the config keys and the common set
 # of arguments that controls the *output* side of things.
 class FFMpegBase(object):
+    '''Mixin class for FFMpeg output
```

Comment:
It's a bit pedantic, but the first line on each of these newly documented classes sometimes does and sometimes doesn't end with a period.